core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/StackV4Controller.java

```diff
@@ -610,4 +610,9 @@ public UsedSubnetsByEnvironmentResponse getUsedSubnetsByEnvironment(Long workspa
                 .stream().map(s -> new UsedSubnetWithResourceResponse(s.getName(), s.getSubnetId(), s.getResourceCrn(), s.getType().name()))
                 .collect(Collectors.toList()));
     }
+
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)
```

Comment:
Is this temporary? You could reuse the already existing right:
http://github.com/hortonworks/cloudbreak/blob/635d404a6448f6fa5a8a216cbd556ca864a462fa/authorization-common-api/src/main/java/com/sequenceiq/authorization/resource/AuthorizationResourceAction.java#L87-L87

Reply:
Ah I was not aware, I will switch to using this.